components/Home/Home.vue

```diff
@@ -48,10 +48,7 @@
           name="headers"
           appear
           mode="out-in"
-          :class="[
-            'tw-hidden md:tw-block',
-            'flex-none tw-max-w-md tw-overflow-scroll',
-          ]"
+          :class="['tw-hidden md:tw-block', 'flex-none tw-max-w-md']"
```

Comment:
Ca you confirm it should not break the vertical scroll ?

Reply:
I've rechecked and it does. will push the new fix